torch/ao/quantization/backend_config/qnnpack.py

```diff
@@ -129,7 +129,6 @@ def get_qnnpack_backend_config() -> BackendConfig:
         qnnpack_default_op_quint8_dtype_config,
     ]
     fixed_qparams_op_dtype_configs = [
-        qnnpack_default_op_qint8_symmetric_dtype_config,
```

Comment:
why this one is removed?

Reply:
Because this dtype config wasn't compatible with the 0to1 and neg1to1 observer constraints (since these are in quint8), though it's technically not required for this PR and I can revert this change